source/jormungandr/jormungandr/external_services/vehicle_occupancy.py

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) 2001-2021, Canal TP and/or its affiliates. All rights reserved.
+#
+# This file is part of Navitia,
+#     the software to build cool stuff with public transport.
+#
+# Hope you'll enjoy and contribute to this project,
+#     powered by Canal TP (www.canaltp.fr).
+# Help us simplify mobility and open public transport:
+#     a non ending quest to the responsive locomotion way of traveling!
+#
+# LICENCE: This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Stay tuned using
+# twitter @navitia
+# channel `#navitia` on riot https://riot.im/app/#/room/#navitia:matrix.org
+# https://groups.google.com/d/forum/navitia
+# www.navitia.io
+
+from __future__ import absolute_import, print_function, unicode_literals, division
+from jormungandr import cache, app
+import pybreaker
+import logging
+import requests as requests
+from six.moves.urllib.parse import urlencode
+from jormungandr.interfaces.v1.serializer.free_floating import FreeFloatingsSerializer
+from jormungandr.external_services.external_service import AbstractExternalService
+
+
+class VehicleOccupancyProvider(AbstractExternalService):
+    """
+    Class managing calls to forseti webservice, providing vehicle_occupancies
+    """
+
+    def __init__(self, service_url, timeout=2, **kwargs):
+        self.logger = logging.getLogger(__name__)
+        self.service_url = service_url
+        self.timeout = timeout
+        self.breaker = pybreaker.CircuitBreaker(
+            fail_max=kwargs.get('circuit_breaker_max_fail', app.config['CIRCUIT_BREAKER_MAX_FORSETI_FAIL']),
+            reset_timeout=kwargs.get(
+                'circuit_breaker_reset_timeout', app.config['CIRCUIT_BREAKER_FORSETI_TIMEOUT_S']
+            ),
+        )
+
+    @cache.memoize(app.config.get(str('default_cache'), {}).get(str('TIMEOUT_FORSETI'), 60 * 60))
```

Comment:
please use `CACHE_CONFIGURATION`, `default_cache` is just an example of cache. `CACHE_CONFIGURATION` is used to deploy.
IMO, a better way to get the default_cache is:
```suggestion
    @cache.memoize(app.config.get(str('CACHE_CONFIGURATION'), app.config.get(str('default_cache')).get(str('TIMEOUT_FORSETI'), 60 * 60))
```
But it's too verbose...